profiles/updates/3Q-2024

```diff
@@ -2,3 +2,4 @@ move sci-libs/rtree dev-python/rtree
 move dev-python/mapbox_earcut dev-python/mapbox-earcut
 move dev-qt/kdsingleapplication dev-libs/kdsingleapplication
 slotmove dev-util/kdevelop-pg-qt 5 0
+move dev-java/ant-ivy dev-java/apache-ivy
```

Comment:
as the recent case we had, the move should be dropped

Reply:
> as the recent case we had, the move should be dropped

done.